Formula/apache-flex-sdk.rb

```diff
@@ -0,0 +1,49 @@
+class ApacheFlexSdk < Formula
+  desc "Framework for building expressive web and mobile applications"
+  homepage "https://flex.apache.org/"
+  url "http://mirror.fibergrid.in/apache/flex/installer/3.2/apache-flex-sdk-installer-3.2.0-src.tar.gz"
+  version "3.2.1"
+  sha256 "f8fb00d300c74609771b23e2bc5d9a97e22f793518245be5596d9a4177236f9e"
+
+  option "with-playerglobal", "Installs PlayerGlobal.swc"
+
+  resource "flash-playerglobal" do
+    url "https://fpdownload.macromedia.com/get/flashplayer/updaters/25/playerglobal25_0.swc"
+    version "25.1"
+  end
+
+  def install
+    Dir["*"].each { |file| cp_r file, File.join(prefix, File.basename(file)) }
+
+    if build.with? "playerglobal"
+      (prefix/"framework/libs/player/25.0").mkpath
```

Comment:
This is unnecessary; this path will be created. You also use `framework` in one place and `frameworks` in the other.